tensorpipe/test/channel/channel_test_cuda.cc

```diff
@@ -120,3 +121,118 @@ class ReceiverWaitsForStartEventTest
 };
 
 CHANNEL_TEST(CudaChannelTestSuite, ReceiverWaitsForStartEvent);
+
+class SendAcrossDevicesTest : public ClientServerChannelTestCase<CudaBuffer> {
+  static constexpr size_t kSize = 1024;
+
+ public:
+  void run(ChannelTestHelper<CudaBuffer>* helper) override {
+    if (TestEnvironment::numCudaDevices() < 2) {
+      GTEST_SKIP() << "Skipping test requiring >=2 CUDA devices.";
+      return;
```

Comment:
Do we need to return? One of the gtest examples seems to hint GTEST_SKIP will somehow skips what comes after (I guess it raises an exception). https://github.com/google/googletest/blob/master/googletest/test/gtest_skip_test.cc#L37-L40

Reply:
Updated.